bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -146,6 +148,9 @@ private final class KeyboardShortcuts {
 	private Color normalTextForegroundColor;
 	private boolean positionAtTop = true;
 	private final TargetPartVisibilityHandler targetPartVisibilityHandler;
+
+	private ControlDecoration searchBarDecoration;
+
```

Comment:
NIT: preserve line-spacing (no extra new lines)
```suggestion
	private ControlDecoration searchBarDecoration;
```

Reply:
done